npm/vue/cypress/component/test-utils-api/TestUtilsApi.vue

```diff
@@ -6,7 +6,10 @@
 <script>
 export default {
   props: {
-    msg: String,
+    msg: {
+      type: String,
```

Comment:
Was this intended?

Reply:
Yeah, this was to resolve one of the warnings that was showing up during linting just to clean some things up. It looks like ESLint requires a default for Vue props the way we have it configured, or else it warns.